File: apps/provider-api/Models/InstitutionLocationAddress.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ProviderApi.Models
{
    public partial class InstitutionLocationAddress
    {
        public int InstitutionLocationAddressId { get; set; }
        public int? InstitutionLocationId { get; set; }
        public Guid? AddressGuid { get; set; }
        public string AddressUsage { get; set; }
        public string AddressType { get; set; }
        public int? ListSequence { get; set; }
        public string AddressLine1 { get; set; }
        public string AddressLine2 { get; set; }
        public string AddressLine3 { get; set; }
        public string AddressLine4 { get; set; }
        public string City { get; set; }
        public string ProvinceState { get; set; }
        public string PostalZipCode { get; set; }
        public string Country { get; set; }
        public DateTime? AddressBusinessStartDate { get; set; }
        public DateTime? AddressBusinessEndDate { get; set; }

        public virtual InstitutionLocation InstitutionLocation { get; set; }
    }
}

File: libs/common/models/institution-location-address.ts


import { InstitutionLocation } from './institution-location';

export class InstitutionLocationAddress {
    public institutionLocationAddressId: number = 0;
    public institutionLocationId: number;
    public addressGuid: string;
    public addressUsage: string = '';
    public addressType: string = '';
    public listSequence: number;
    public addressLine1: string = '';
    public addressLine2: string = '';
    public addressLine3: string = '';
    public addressLine4: string = '';
    public city: string = '';
    public provinceState: string = '';
    public postalZipCode: string = '';
    public country: string = '';
    public addressBusinessStartDate: Date;
    public addressBusinessEndDate: Date;
    public institutionLocation: InstitutionLocation = new InstitutionLocation();

    constructor(init? : Partial<InstitutionLocationAddress>) {
        
        Object.assign(this, init);
    }
}

